src/main.rs

```diff
@@ -246,6 +246,14 @@ fn parse_matches() -> clap::ArgMatches {
                 .default_value(&DEFAULT_INTERNAL_NODE_STAKE_SOL.to_string())
                 .help("Amount to stake internal nodes (Sol)."),
         )
+        .arg(
+            Arg::with_name("commission")
+                .long("commission")
+                .value_name("PERCENTAGE")
+                .takes_value(true)
+                .default_value("100")
+                .help("The commission taken by nodes on staking rewards (0-100) [default: 100]")
```

Comment:
```suggestion
                .help("The commission taken by nodes on staking rewards (0-100)")
```

Reply:
Don't need to write out [default ...] since clap's `default_value()` will add that in automatically

Reply:
Oh, nice! Must be a newer-clap thing